### PR TITLE
[example] fix inset activity MAPSMBL-156

### PR DESCRIPTION
**Pull request by ank27**
### Summary of changes
 - Fixed crash when `insetMap` is not initialised.
 - Update insetFragment to set style on configuration change.

## Pull request checklist:
 - [ ] Briefly describe the changes in this PR.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
    
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [ ] Optimize code for java consumption (`@JvmOverloads`, `@file:JvmName`, etc).
 - [ ] Add example if relevant.
 - [ ] Document any changes to public APIs.
 - [ ] Run `make update-api` to update generated api files, if there's public API changes, otherwise the `verify-api-*` CI steps might fail.
 - [ ] Update [CHANGELOG.md](../CHANGELOG.md) or use the label 'skip changelog', otherwise `check changelog` CI step will fail.
 - [ ] If this PR is a `v10.[version]` release branch fix / enhancement, merge it to `main` firstly and then port to `v10.[version]` release branch.

Fixes: < Link to related issues that will be fixed by this pull request, if they exist >

PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-android/blob/main/CONTRIBUTING.md#contributor-license-agreement).

### CODE REVIEW

File: app/src/main/java/com/mapbox/maps/testapp/examples/InsetMapActivity.kt [hunk 1]
```diff
@@ -110,8 +114,10 @@ class InsetMapActivity : AppCompatActivity(), OnCameraChangeListener {
       .bearing(mainCameraPosition.bearing)
       .center(mainCameraPosition.center)
       .build()
-    insetMapboxMap.setCamera(insetCameraPosition)
-    insetMapboxMap.getStyle { style -> updateInsetMapLineLayerBounds(style) }
+    if (::insetMapboxMap.isInitialized) {
```
Comment (kiryldz) [inline app/src/main/java/com/mapbox/maps/testapp/examples/InsetMapActivity.kt:117]: perhaps make nullable instead?

**Review by kiryldz — APPROVED** [review]
LGTM with a question, main thing that issues are fixed 😄

File: app/src/main/java/com/mapbox/maps/testapp/examples/InsetMapActivity.kt (hunk 1 above)
Comment (ank27) [reply app/src/main/java/com/mapbox/maps/testapp/examples/InsetMapActivity.kt]: done

File: app/src/main/java/com/mapbox/maps/testapp/examples/InsetMapActivity.kt [hunk 2]
```diff
@@ -51,57 +51,61 @@ class InsetMapActivity : AppCompatActivity(), OnCameraChangeListener {
...
-          }
-        }
-      }
-
+      setInsetMapStyle(insetMapFragment)
       // Add fragmentMap fragment to parent container
       transaction.add(R.id.mini_map_fragment_container, insetMapFragment, INSET_FRAGMENT_TAG)
       transaction.commit()
+    } else {
+      setInsetMapStyle(insetMapFragment)
```
Comment (pengdev) [inline app/src/main/java/com/mapbox/maps/testapp/examples/InsetMapActivity.kt:60]: we can simply move this line out of the if else logic? as both condition will execute the `setInsetMapStyle`

**Review by pengdev — APPROVED** [review]
LGTM with 1 nit

File: app/src/main/java/com/mapbox/maps/testapp/examples/InsetMapActivity.kt (hunk 2 above)
Comment (ank27) [reply app/src/main/java/com/mapbox/maps/testapp/examples/InsetMapActivity.kt:60]: yeah correct 😄

**Review by yunikkk — APPROVED** [review]

**Pull request merged**